src/components/appHeader/login.jsx

```diff
@@ -1,22 +1,26 @@
-import React, {useState} from 'react'
+import React, {useState, useContext} from 'react'
 import { Card, Row, Button } from 'antd'
 import { useHistory } from 'react-router-dom'
 import './styles.css'
 import {handleLogin, resetPassword} from "./apiCalls";
 import AntdInput, {AntdInputPassword} from "../antdMappedComponents/antdMapper";
+import { UserContext } from "../../App";
+
 
 const Login = () => {
     const [email, setEmail] = useState('')
     const [password, setPassword] = useState('')
     const [error, setErrors] = useState('')
     const history = useHistory();
+    
+    const { setUserInfo } = useContext(UserContext)
```

Comment:
Does this work? Not sure how it can with setUserInfo being passed in AuthContext

Reply:
Updated to be passed though its own provider